pkg/config/config.go

```diff
@@ -475,7 +478,8 @@ func DefaultConfig() (*Config, error) {
 			GRPCMaxRecvMsgSize: defaultGRPCMaxMsgSize,
 		},
 		RuntimeConfig: RuntimeConfig{
-			DefaultRuntime: defaultRuntime,
+			DecryptionKeysPath: "/etc/crio/keys/",
```

Comment:
Do we want to have a more general location here within `/etc/containers/...`?

Reply:
I am not sure about `/etc/containers`. It seems to me that the directory holds various configurations that are applicable to the broader `containers/image` scope. 

```sh
# ls -l /etc/containers/* 
-rw-r--r-- 1 root root  256 Aug 10 21:39 /etc/containers/policy.json
-rw-r--r-- 1 root root  881 Sep 30 08:49 /etc/containers/registries.conf
-rw-r--r-- 1 root root  891 Aug 10 21:39 /etc/containers/storage.conf

/etc/containers/registries.d:
total 4
-rw-r--r-- 1 root root 886 Aug 10 21:39 default.yaml
``` 

While the main purpose of the path `/etc/crio/keys` is to store the private keys (not conf files) needed only by `CRI-O` to decrypt images.

Reply:
Makes sense :+1: